conf/app.routes

```diff
@@ -1,5 +1,7 @@
 # microservice specific routes
 
+->         /hmrc-frontend           hmrcfrontend.Routes
```

Comment:
Is that routing file auto-magically inherited?

Reply:
Yes - its part of `play-frontend-hmrc`
https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/resources/hmrcfrontend.routes